alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/accessor/UniqueConfigurationAccessor.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * alert-common
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.common.persistence.accessor;
+
+import java.util.Optional;
+
+import com.synopsys.integration.alert.api.common.model.AlertSerializableModel;
+import com.synopsys.integration.alert.api.common.model.exception.AlertConfigurationException;
+
+public interface UniqueConfigurationAccessor<T extends AlertSerializableModel> {
+    String DEFAULT_CONFIGURATION_NAME = "default-configuration";
+
+    Optional<T> getConfiguration();
+
+    Optional<T> getConfigurationByName(String configurationName);
```

Comment:
You don't need this method.  If this is a uniqueAccessor then you don't need to pass the configuration name.  You just need getConfiguration.  The implementer can hide that it looks up 'default-configuration'